website/templates/project/settings.mako

```diff
@@ -377,6 +387,8 @@
     <script>
       window.contextVars = window.contextVars || {};
       window.contextVars.node = window.contextVars.node || {};
+      window.contextVars.node.description = '${node['description']}';
```

Comment:
Try to avoid building strings in a template this way- if the description or url contained an apostrophe halfway through the string, the resulting malformed javascript would break the entire page. (`'${pythonvar}'` is not recommended)

Instead, when you're inserting mako variables into a <script> block, use the `| sjson, n` pair of mako filters (and don't enclose these values in single quotes). 

See examples for `nodeType` and `nodeCategories` a few lines below.